presto-hive-hadoop2/bin/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+cd "${BASH_SOURCE%/*}/../.."
```

Comment:
Why do we need this?

Reply:
This way, all `test.sh` scripts:
 - have same working directory
 - their content is copyable, so developer can run easily run tests manually
 - I didn't have to do any changes to test commands when extracting them from `.travis.yml`.